examples/abi/README.md

```diff
@@ -0,0 +1,9 @@
+# ABI
+
+Showcases how to generate ABI from a NEAR contract.
+
+## Generating ABI
+To test run:
+```bash
+cargo near metadata
```

Comment:
Should this not be something like `cargo near abi` or something? Feel like the metadata naming might be confusing